packages/xs-extra/xapi.master/opam

```diff
@@ -43,6 +43,7 @@ depends: [
   "xapi-rrdd-plugin"
   "pci"
   "sha"
+  "xenopsd"
```

Comment:
Will we need to add `ulimit -s unlimited` or `OCAMLRUNPARAMS` to avoid segfaulting on the datamodel?